material-fab/src/main/ets/components/MainPage/fabUtils/fabCommonProperties.ets

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RippleModel } from '../ripple/rippleModel'
+
+const DEFAULT_FAB_ICON: ResourceStr = undefined
```

Comment:
check for validations at their usage

Reply:
changed value of default variables from undefined to some other value
only DEFAULT_FAB_ICON is undefined for now and validations have been added at it's usage